spacialgaze-plugins/misc.js

```diff
@@ -7,6 +7,29 @@
  * @license MIT license
  */
 'use strict';
+'use strict';
```

Comment:
Ok, there is no way in the entirety of the world where you should have 2 `'use strict';`